### ISSUE TITLE
feat: Add Custom Webhook Support for Automation Actions

### ISSUE THREAD

**Issue by zehonghuang** [apitable/apitable#1477]
## Description
We got automation for Slack and stuff. But we need something for more things, not just Slack. Want to add a thing for custom webhooks in automation.

## Functionality
We need to set up their own webhooks and can put in the webhook URL, choose if it's a GET or POST, and write what goes in the header and body.

**Comment by xukecheng** [comment]
It's great to receive your feedback, could you please elaborate on your workflow and specific scenarios? This will be very helpful for our project creation, product design and development.

**Comment by fangjiyuan** [comment]
This is what i want to say.  
I found in develop cook url, there is a api may be can  do this .
https://apitable.getoutline.com/s/82e078fc-1a8d-4616-b69d-fcdbb18ef715/doc/automation-5khWD8n9Gz
/api/v1/automation/service/create 
I try to run but false.

**Comment by zehonghuang** [comment]
This feature is similar to the custom webhook of Prometheus' Alertmanager. The interface only needs to receive a fixed data structure.
@xukecheng

**Comment by xukecheng** [comment]
> This feature is similar to the custom webhook of Prometheus' Alertmanager. The interface only needs to receive a fixed data structure. @xukecheng

Yes, I totally understand what you are saying, but in order to get a clearer picture of your needs, could you please describe the specifics, e.g. could you share a story or example of how you have used this feature? What problems have you encountered that you would like to solve with this feature?

**Comment by mr-kelly** [comment]
Thank you for the detailed context regarding the Prometheus Alertmanager use case in the comments! Allowing Custom Webhooks (defining URLs, methods, headers, and payload structure) as an Automation Action is indeed a very powerful feature for integrating with external systems. We have recorded this enhancement and passed the use-case details to our product team for evaluation.
